scripts/apitypings/main.go

```diff
@@ -67,11 +68,6 @@ type TypescriptTypes struct {
 // String just combines all the codeblocks.
 func (t TypescriptTypes) String() string {
 	var s strings.Builder
-	const prelude = `
-// Code generated by 'make site/src/api/typesGenerated.ts'. DO NOT EDIT.
-
-`
-	_, _ = s.WriteString(prelude)
 
```

Comment:
Should we still have a prelude for the source? Like `This section comes from codersdk` or something?

A divider would help make sense of the sorting imo